src/main/java/edu/wpi/grip/ui/Main.java

```diff
@@ -62,5 +65,11 @@ public final void onFatalErrorEvent(FatalErrorEvent error) {
                 }
             }
         });
+
+        if(event.isFatal()) {
+            System.err.println("Original fatal exception");
+            event.getThrowable().printStackTrace();
```

Comment:
We already print the stack trace [here](https://github.com/WPIRoboticsProjects/GRIP/pull/179/files#diff-d5c3d322c479e7a4fc933fa793c49bcdR52), right?